README.md

```diff
@@ -31,10 +31,50 @@ For a detailed example please clone the project and look at the included sample
 
 ### The Basics
 
-Firstly you'll need some to save your GraphQL queries and mutations into .graphql files, You can use this tool to generate your Insomnia workspaces into directories and files [insomnia-graphql-generator](https://github.com/AniTrend/insomnia-graphql-generator) and place these files into your assets folder as shown below:
+First, you'll need .graphql files for saving your GraphQL queries, fragments, and mutations. You can use this tool to generate your Insomnia workspaces into directories and files [insomnia-graphql-generator](https://github.com/AniTrend/insomnia-graphql-generator), placing them into your assets folder as shown below:
```

Comment:
Documentation updated to call out the new support for externalized fragments.

Reply:
Really appreciate the effort to fix up and update the documentation :100: